src/modules/menu/components/advancedTools/ImportFile/ImportFileOnlineTab.vue

```diff
@@ -76,6 +79,13 @@ async function loadFile() {
 
     try {
         await handleFileSource(fileUrl.value, false)
+        if (!fileUrl.value.match(/https:\/\//)) {
```

Comment:
i don't think this will catch urls like http://foo.com?url=https://bar.com
maybe something like `\G/https:\/\//` could work

Reply:
ho yeah, good catch :)